### PR TITLE
Add Eurostat NUTS concordances and props

### PR DESCRIPTION
**Pull request by nvkelso**
Related to https://github.com/whosonfirst-data/whosonfirst-data/issues/2133

### CODE REVIEW

File: data/856/331/59/85633159.geojson [hunk 1]
```diff
@@ -4,8 +4,16 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "eurostat:country_code":"UK",
```
Comment (stepps00) [inline data/856/331/59/85633159.geojson:7]: Confirming WOF maintains a source for this: https://github.com/whosonfirst/whosonfirst-sources/blob/main/sources/README.md#eurostat

File: data/856/331/59/85633159.geojson [hunk 2]
```diff
@@ -4,8 +4,16 @@
...
+    "eurostat:level_code":"0",
+    "eurostat:name_latin":"United Kingdom",
+    "eurostat:name_local":"United Kingdom",
```
Comment (stepps00) [inline data/856/331/59/85633159.geojson:10]: Probably not for this PR set, but would be harvest these names for new/updated `name:` properties.

**Review by stepps00 — APPROVED** [review]
LGTM

File: data/856/331/59/85633159.geojson (hunk 2 above)
Comment (nvkelso) [reply data/856/331/59/85633159.geojson:10]: Filed https://github.com/whosonfirst-data/whosonfirst-data/issues/2139

**Pull request merged**